examples/docker.yaml

```diff
@@ -7,6 +7,7 @@
 # $ docker ...
 
 # This example requires Lima v0.8.0 or later
+driver: "vz"
```

Comment:
The default must remain qemu

Reply:
You can create `examples/vz.yaml` for the VZ example. (Maybe under `experimental` dir?)

Reply:
Yes yes that make sense. Will put it under experimental directory.

Reply:
Done.